src/auth/auth.proto

```diff
@@ -86,6 +86,8 @@ message TokenInfo {
     GET_TOKEN = 2;  // returned by GetToken()--revokeable.
   }
   TokenSource source = 2;
+
+  string hash = 3;
```

Comment:
Is this relevant? Is this just because there was work done on master?

Reply:
I made a slight change to the collection interface that we no longer return the key in various callbacks - individual collections are expected to be able do derive the key from the stored protobuf (if they need it).  This might be a bad idea and maybe I should revert this.

Reply:
If every collection has a primary key then I would say revert.  But if they don't have a primary key conceptually, and we just choose one in postgres then this is a better fit for the model.